frontend/lib/src/components/elements/ArrowVegaLiteChart/CustomTheme.tsx

```diff
@@ -89,13 +89,13 @@ export function applyStreamlitTheme(config: any, theme: EmotionTheme): any {
       titleFontWeight: theme.fontWeights.normal,
       titleFontStyle: "normal",
       titleColor: getGray70(theme),
-      titlePadding: 5,
+      titlePadding: convertRemToPx("0.313rem"),
```

Comment:
We probably want to change these values to align better and reuse one of our spacing values. But that will be a  dedicated project.